packages/teams-js/src/artifactsForCDN/validDomains.json

```diff
@@ -37,6 +37,7 @@
     "fa000000125.resources.office.net",
     "fa000000129.resources.office.net",
     "fa000000124.resources.office.net",
-    "fa000000128.resources.office.net"
+    "fa000000128.resources.office.net",
+    "*.cloud.microsoft"
```

Comment:
Just curious, will this cover from line 32-34?

Reply:
The answer is yes, but I think it's important for us to also distinguish the explicit partner requirements if we ever need to remove the wildcard